decidim-admin/app/packs/stylesheets/decidim/admin/_item_edit.scss

```diff
@@ -75,12 +71,6 @@
       .editor-container .editor-input .ProseMirror {
         @apply bg-background-2;
       }
-
-      &.edit_content_block {
```

Comment:
After removing this, it seems there is an error with the spacing between fields (the field is too close to the next label). Additionally, i see the border of the "add file" is busted.  

![image](https://github.com/user-attachments/assets/7fb1f77f-cc70-4eb4-bc74-0c9439682fa7)